src/hooks.ts

```diff
@@ -342,23 +343,28 @@ export const useFeature: UseFeature = (featureKey, options = {}, overrides = {})
  *       ClientReady and DidTimeout provide signals to handle this scenario.
  */
 export const useDecision: UseDecision = (flagKey, options = {}, overrides = {}) => {
+  const [lastUserUpdate, setLastUserUpdate] = useState<Date | null>(null);
+  const store = clientStore.getInstance();
   const { optimizely, isServerSide, timeout } = useContext(OptimizelyContext);
   if (!optimizely) {
     throw new Error('optimizely prop must be supplied via a parent <OptimizelyProvider>');
   }
 
   const overrideAttrs = useCompareAttrsMemoize(overrides.overrideAttributes);
-  const getCurrentDecision: () => { decision: OptimizelyDecision } = useCallback(
-    () => ({
-      decision: optimizely.decide(flagKey, options.decideOptions, overrides.overrideUserId, overrideAttrs)
-    }),
-    [optimizely, flagKey, overrides.overrideUserId, overrideAttrs, options.decideOptions]
-  );
+  const getCurrentDecision: () => { decision: OptimizelyDecision } = () => ({
+    decision: optimizely.decide(flagKey, options.decideOptions, overrides.overrideUserId, overrideAttrs),
+  });
```

Comment:
This was probably done to make sure `decide` is only called if any of the inputs that it depends on, change. Removing this from a `useCallback` hook might result in it being called multiple times on same inputs